index.html

```diff
@@ -1,10 +1,138 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
+<html lang="en" xmlns="http://www.w3.org/1999/html">
+  <h1>Моя первая страница</h1>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="description" content="IT school from Belarus" />
+    <meta name="keywords" content="HTML,CSS,Javascript,React" />
+    <meta name="author" content="Irina Breiksh" />
+    <meta name="viewport" content="width=device-width,initial-scale=1.0" />
     <title>Title</title>
-</head>
-<body>
-
-</body>
-</html>
\ No newline at end of file
+  </head>
+  <body>
+    <hr />
+    <img src="./images/matrix.jpg" alt="coding matrix" />
+    <ol>
+      <li>Знакомство с Web
+        <ul>
+          <li>Краткая история Web и как устроен интернет</li>
+          <li>Как устроен браузер и как он получает страницы по сети</li>
+          <li>Что такое Git</li>
+          <li>Создание первого репозитория</li>
+          <li>О файле .gitignore</li>
+        </ul>
+      </li>
+      <li>HTML-вёрстка – первые шаги
+        <ul>
+        <li>Настройка рабочей среды</li>
+        <li>Краткая история HTML</li>
+        <li>Типы тегов и атрибуты</li>
+        <li>Теги для работы с текстом</li>
+        <li>Создание собственной страницы</li>
+        <li>Основы ветвления в Git</li>
+        <li>Создание pull request на GitHub</li>
+      </ul>
+      </li>
+      <li>Погружение в HTML-разметку
+        <ul>
+        <li>Мета-теги</li>
+        <li>Мнемоники</li>
+        <li>Теги для работы со списками</li>
+        <li>Теги для работы с таблицами</li>
+        <li>Теги для работы с формами</li>
+        <li>div и span</li>
+      </ul>
+      </li>
+    </ol>
+    <table border="1" width="50%">
+      <thead>
+      <tr>
+        <th>Tag</th>
+        <th>Description</th>
+      </tr>
+      </thead>
+      <tbody>
+      <tr>
+        <td>&#60table&#62</td>
+        <td>Defines a table</td>
+      </tr>
+      <tr>
+        <td>&#60tr&#62</td>
+        <td>Defines a row in a table</td>
+      </tr>
+      <tr>
+        <td>&#60th&#62</td>
+        <td rowspan="2">Defines a &#34header cell&#34 &#8260 &#34cell&#34 in a table</td>
+      </tr>
+      <tr>
+        <td>&#60td&#62</td>
+      </tr>
+      <tr>
+        <td colspan="2">Rarely used</td>
+      </tr>
+      <tr>
+        <td>&#60caption&#62</td>
+        <td>Defines a table caption</td>
+      </tr>
+      <tr>
+        <td>&#60thead&#62</td>
+        <td>Groups the header content in a table</td>
+      </tr>
+      <tr>
+        <td>&#60tbody&#62</td>
+        <td>Groups the body content in a table</td>
+      </tr>
+      <tr>
+        <td>&#60tfoot&#62</td>
+        <td>Groups the foot content in a table</td>
+      </tr>
+      </tbody>
+<tfoot>
+</tfoot>
```

Comment:
You may not use the optional tags if there is no content.

Reply:
Ok